alephmarcreader/abstractalephmarcreader.py

```diff
@@ -353,4 +362,117 @@ def get_recipient_organisation(self):
 
         return recipient_organisation
 
-    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
\ No newline at end of file
+    get_recipient_organisation.__annotations__ = {'return': [Organisation]}
+
+    def get_supplement_remarks(self):
+        """
+        Returns comments on supplementing material following the letter. (See also Description.supplement.)
```

Comment:
How is this related to `Description.supplement`?

Reply:
Description.supplement (or Field 300, Subfield e) lists all "Begleitmaterial", the supplement remarks (Field 525) explains, what the Begleitmaterial is.

I was hoping we could get rid of one of these things, but aparently they're quite different. E.g.: 000054780 
- 300e could be something like `Beilage (3 Bl. (4, 4a, 4b))`
- 525 could be something like `Dem Brief liegt als fo. 4 (24 x 19 cm), fo. 4a (23,5 x 18,5 cm) und fo. 4b (12 x 18,5 cm) eine autographe Frühfassung von Daniel Bernoullis Abhandlung "Specimen theoriae novae de mensura sortis" bei. Diese unterscheidet sich von der Druckfassung in: CP V (1730/31) pp.175-192 (St.22).`

Reply:
I was just confused to find a reference to a type that is not returned by this method.

So if these fields are semantically related, they should probably be combined into one method. Otherwise, keep them separate but do not cross reference.

Reply:
They are semantically related in the sense that they both refer to the supplement; but 300e goes better with physical description than with 525.  
I'll remove the reference then.

Reply:
Ok